odysseus/convertFtbQuests.ts

```diff
@@ -383,6 +383,21 @@ function truncateLong(value: Long | undefined) {
     return Number(value)
 }
 
+function iconBackgroundTexture(iconBackground: QuestShape | undefined) {
+	switch (iconBackground) {
```

Comment:
At this point, why not just return `heracles:textures/gui/quest_backgrounds/${iconBackground}s.png`?

Reply:
Because I know nothing about typescript, ^^.
Will make that change, it looks much better. Thank you for pointing it out.

Reply:
Heya! Just touched up 101 on heracles to include all shapes now - the only special case you need is a 

`if (iconBackground == 'square') return "heracles:textures/gui/quest_backgrounds/default.png"`

then otherwise you can return as above - i renamed to `rsquares` etc.

Reply:
Um... I just spent a few hours bashing my head on TypeScript. (It's ok... not every lang fits with every coder) to get this to work... mostly on trying to get my patch back down to the one line it where imageBackground was being written before, maybe two.

Apologies, I'm giving up on this particular patch.
If others want to keep it in, I'm happy to do whatever I can git/process-wise to enable that... be it keeping this PR open and accepting edits, or closing this one, or whatever.